gateways.json

```diff
@@ -1,4 +1,5 @@
 [
+  "https://ipfs.cyou",
```

Comment:
@smurfix I think you need to fix TLS setup. Currently, you have wildcard at wrong place:

> ![2024-06-27_22-22](https://github.com/ipfs/public-gateway-checker/assets/157609/16a7202d-a97b-41ab-9424-b352ad6ec86c)

TLS wildcards work only for one level up. Due to this, you need to set up additional TLS wildcard certificates for DNS Names one subdomain higher: `*.ipfs.ipfs.cyou` and `*.ipns.ipfs.cyou`.

See https://docs.ipfs.tech/how-to/address-ipfs-on-web/#subdomain-gateway

You can reproduce with below:

- https://ipfs.cyou/ipfs/bafkreibm6jg3ux5qumhcn2b3flc3tyu6dmlb4xa7u5bf44yegnrjhc4yeq redirects to https://bafkreibm6jg3ux5qumhcn2b3flc3tyu6dmlb4xa7u5bf44yegnrjhc4yeq.ipfs.ipfs.cyou which then fails due to TLS error
- https://ipfs.cyou/ipns/en.wikipedia-on-ipfs.org redirects to https://en-wikipedia--on--ipfs-org.ipns.ipfs.cyou which then fails due to TLS error

Reply:
I'm already working on it. Unfortunately my DNS secondaries tend not to listen to update notifications.

Reply:
Fixed, now uses working wildcard certificates.